types/Code.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const GraphQL = require('graphql');
+
+module.exports = (field) => ({
+	type: GraphQL.GraphQLString,
+	args: {
```

Comment:
These aren't necessary - they're for configuring the field UI, not the formatting of the output